### DISCUSSION TITLE
Consider alternatives to npx and yarn

### DISCUSSION THREAD

**Discussion by AddictArts** [onejs/one#205] [General]
Unable to use npx and yarn. Hopefully there is the possibility to support more package managers and installation methods.

**Comment by natew** [comment]
What do you mean? We support bun and pnpm too so long as you don’t hoist dependencies.

**Comment by AddictArts** [comment]
Thanks for the reply @natew. I think that information is not clear or buried, glad to hear it does. Launching with `npx` also implies `npm` for example. Additionally the source repo uses `yarn`, which could discourage any involvement.

**Comment by natew** [comment]
Our cli prompts you to use any of them but but yea we can keep improving here. I'll move this to a discussion so we can keep chatting about improvements, any actual breaking things we can open issue for.

The site does show "npx" since anyone with node has it, but we could add a switcher in more places (the docs do have a switcher, just not the homepage).

We should also probably detect bad configuration in pnpm/bun for hoisting and throw an error.

**Reply by astriskit** [reply]
Hi there. Any thing in [deno space](https://github.com/onejs/one/issues/198); Thanks.
